src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/index.js

```diff
@@ -3,7 +3,18 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import App from './App';
+////#if (IndividualLocalAuth)
+// Uncomment the lines below to register the generated service worker.
+// By default create-react-app includes a service worker to improve the
+// performance of the application by caching static assets. This service
+// worker, can however try and handle paths for the identity UI, so it is
+// disabled by default when Identity is being used.
+//
+//import registerServiceWorker from './registerServiceWorker';
+////#else
```

Comment:
This causes issues when used along with Identity (only on deployed projects, localhost works fine) so we are disabling it. Users can turn it on and configure it appropriately. Seems like in the future CRA will allow for customizing the service worker in some way.